php/class-plugin.php

```diff
@@ -127,21 +127,21 @@ public function __construct() {
 	 * that extend the Customizer to ensure resources are available in time.
 	 */
 	public function init() {
-
 		$this->components['connect']      = new Connect( $this );
 		$this->components['deactivation'] = new Deactivation( $this );
 		$this->components['sync']         = new Sync( $this );
 		$this->components['media']        = new Media( $this );
 		$this->components['api']          = new REST_API( $this );
 		$this->components['storage']      = new Storage( $this );
+		$this->components['support']      = new Support( $this );
 	}
 
 	/**
 	 * Get a plugin component.
 	 *
 	 * @param mixed $component The component.
 	 *
-	 * @return Connect|Media|REST_API|Settings_Page|Sync|null
+	 * @return Logger|Connect|Media|REST_API|Settings_Page|Sync|null
```

Comment:
@DavidCramer 

I think there's a typo here. Isn't that supposed to be `Support`?
I see many other instances of Logger reference, especially on docblocks. Let's fix those as well ;)

Reply:
@pereirinha - Ah, good catch! That was the original class name, which I changed, but forgot to change the docs.